hasura/migrations/carnet_de_bord/1646132441749_alter_table_public_notebook_target_add_column_status/down.sql

```diff
@@ -0,0 +1,4 @@
+-- Could not auto-generate a down migration.
+-- Please write an appropriate down migration for the SQL below:
+-- alter table "public"."notebook_target" add column "status" varchar
+--  not null default 'in_progress';
```

Comment:
question: on sait pourquoi il a pas réussi à générer ça ? Ça paraît pas fou à faire, si ?
```suggestion
alter table "public"."notebook_target drop column "status";
```